{"name": "test_cart_checkout_button", "status": "failed", "statusDetails": {"message": "AssertionError: Failed to navigate to cart or checkout button not visible.\nassert False\n +  where False = navigate_to_cart(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b10ce7e2dab1033198c28d8a490a968\")>, 'http://127.0.0.1:5000')", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b10ce7e2dab1033198c28d8a490a968\")>\n\n    @pytest.mark.cart_checkout  # Custom marker to group/cart this test\n    def test_cart_checkout_button(driver):\n        base_url = \"http://127.0.0.1:5000\"\n    \n        # Login to the application\n        login_to_flask_shop(driver, base_url, \"abc@gmail.com\", \"abcd@123\")\n    \n        # Navigate to the cart and check if the checkout button is visible\n        try:\n            assert navigate_to_cart(driver, base_url), \"Failed to navigate to cart or checkout button not visible.\"\n        except AssertionError as e:\n            take_screenshot(driver, \"test_cart_checkout_button\")  # Call take_screenshot on failure\n>           raise e  # Re-raise the assertion error after taking screenshot\n\ntest_checkout.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b10ce7e2dab1033198c28d8a490a968\")>\n\n    @pytest.mark.cart_checkout  # Custom marker to group/cart this test\n    def test_cart_checkout_button(driver):\n        base_url = \"http://127.0.0.1:5000\"\n    \n        # Login to the application\n        login_to_flask_shop(driver, base_url, \"abc@gmail.com\", \"abcd@123\")\n    \n        # Navigate to the cart and check if the checkout button is visible\n        try:\n>           assert navigate_to_cart(driver, base_url), \"Failed to navigate to cart or checkout button not visible.\"\nE           AssertionError: Failed to navigate to cart or checkout button not visible.\nE           assert False\nE            +  where False = navigate_to_cart(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"2b10ce7e2dab1033198c28d8a490a968\")>, 'http://127.0.0.1:5000')\n\ntest_checkout.py:32: AssertionError"}, "attachments": [{"name": "stdout", "source": "e58c7afd-1619-4608-bb31-8325b337ab39-attachment.txt", "type": "text/plain"}], "start": 1745079730804, "stop": 1745079758986, "uuid": "6822d3de-d565-4808-adfd-6a90833fb937", "historyId": "db3d56adc82778d80e6f628663ec778f", "testCaseId": "db3d56adc82778d80e6f628663ec778f", "fullName": "test_checkout#test_cart_checkout_button", "labels": [{"name": "tag", "value": "cart_checkout"}, {"name": "suite", "value": "test_checkout"}, {"name": "host", "value": "DESKTOP-H49NC95"}, {"name": "thread", "value": "15576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_checkout"}]}